Objects/StrategyCardAutomator.ttslua

```diff
@@ -822,14 +950,35 @@ function _addTechnologyPrimarySecondary(layout)
 	end
 
 	for _,o in pairs(layout[technologyIndex]["children"]) do
-		 if o["attributes"]["id"] == "technology_secondary" then
-			 return layout
-		 end
+		if o["attributes"]["id"] == "technology_secondary" then
+			return layout
+		end
 	end
-		-- add the Assign Speaker Button which opens the Popup
+
+	if not _setupHelper.getPoK() then
+		-- remove pok tech from ui
+
+		local techlist = {["greenTechs"] = true, ["yellowTechs"] = true, ["redTechs"] = true, ["blueTechs"] = true}
+		local horizontalKeep = {}
+		for _,o in pairs(layout[technologyIndex]["children"]) do
+			local counter = 1
+			if o["tag"] == "HorizontalLayout" and techlist[o["attributes"]["id"]] then
+				for iv,v in ipairs(o["children"]) do
```

Comment:
Hard coding these positions is fine, but fragile should the tech xml ever change.  I don't think it's worth the hassle of removing by name (because if the XML changes who knows what the layout is).